domains/sameerahmed.json

```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "sxmeer-ahmed",
+        "email": "ahmedkhansameer50@gmail.com"
+      },
+      "record": {
+        "TXT": "786"
```

Comment:
Please provide a valid record

Reply:
> Please provide a valid record

It is a valid record.

Reply:
> > Please provide a valid record
> 
> It is a valid record.

It isn't as it is not used for anything and it's their only record.

Reply:
We don't know what he could be using it for, its valid as in DNS and he could be doing something of his own with that record.

Reply:
> We don't know what he could be using it for, its valid as in DNS and he could be doing something of his own with that record.

We have to know what he's using it for, it's literally in the docs.